### PR TITLE
add passed test result to report

### PR DESCRIPTION
**Pull request by oouyang**

### CODE REVIEW

**Comment by askeing** [comment]
we can remove ` import time` and `import zipfile` from `certsuite/harness.py`.

File: certsuite/report/report.css [hunk 1]
```diff
@@ -20,6 +20,7 @@ body {
 }
 .PASS { background-color: green }
 .FAIL { background-color: red }
+.OK { background-color: lightgreen}
```
Comment (altsai) [inline certsuite/report/report.css]: I'll suggest to use "EXECUTED" for bringing correct images. "OK" sounds like it get passed.

Comment (oouyang) [inline certsuite/report/report.css]: This 'OK' is the current tag in code, it only shows in cert test for parent test. I suggest let it be OK now and adjust it to right tag after considering all tags to wrap to new tag name.

Comment (altsai) [inline certsuite/report/report.css]: Go it. let's merge the patch.

**Pull request merged**